src/Cron_Event_Command.php

```diff
@@ -15,7 +15,7 @@
  *     $ wp cron event run --due-now
  *     Success: Executed a total of 2 cron events.
  *
- *     # Delete the next scheduled cron event
+ *     # Delete all scheduled cron events for the given hook
```

Comment:
```suggestion
 *     # Deletes all scheduled cron events for the given hook
```
Should it be `Deletes` based on https://github.com/wp-cli/wp-cli/issues/4542?

Reply:
Don't you like the standard imperative??

Reply:
Just a suggestion based on [existing documentation ](https://github.com/wp-cli/cron-command/pull/62/files#diff-6c350b5369d7bf7329ce3502ab730900R347)🙂 sorry if incorrect. I'll wait for @johnbillion and @schlessera to take a final decision.

Reply:
This form would be required by WPCS for the docblocks. But here, we describe what the example does, and we've been using the standard imperative for that.